deps/chakrashim/src/jsrtinspectorhelpers.cc

```diff
@@ -290,15 +335,59 @@ namespace jsrt {
                             &JsConvertValueToString, wasCopied);
   }
 
+  JsErrorCode InspectorHelpers::ArrayConcat(JsValueRef array, JsValueRef value) {
```

Comment:
>InspectorHelpers [](start = 14, length = 16)

Could you have use jsrtutils as much as possible and write common code in it instead of InspectorHelpers?